File: Arrays/pairsInArray.py
arr = [12, 3, 4, 55, 6]
count = 0
for i in range(0, len(arr) - 2):
    for j in range(i + 1, len(arr)):
        count += 1
        print((arr[i], arr[j]), end =" ")
    print()
print(count)

File: Arrays/pairsInArray.java
public class pairsInArray{
    public static void main(String[] args){
        int[] arr = {12, 3, 4, 55, 6};
        pairs(arr);
    }
    public static void pairs(int[] arr){
        for(int i = 0; i < arr.length - 2; i++){
            for(int j = i + 1; j < arr.length; j++){
                System.out.print("(" + arr[i] +", " + arr[j] + ") ");
            }
            System.out.println();
        }
    }
}